tsdb/block.go

```diff
@@ -124,6 +124,9 @@ type BlockReader interface {
 
 	// Meta provides meta information about the block reader.
 	Meta() BlockMeta
+
+	// Size returns the number of bytes that the block takes up.
```

Comment:
```suggestion
	// Size returns the number of bytes that the block takes up on disk.
```

Reply:
In the interface, it might no be disk, but e.g. s3.

Reply:
The size function will also be implemented by the Head object, which does not use disk, right?

Reply:
Ignore that my last comment, I've read again ganesh's blog post about prometheus Head... I see that it uses disk for wal and completed chunks